tools/api-inventory/scripts/detect_routes.py

```
@@ -0,0 +1,600 @@
+    """先頭 `/api` の有無を吸収した比較キー。"""
+    u = norm_uri(u)
+    out = {u}
+    if u.startswith('/api'):
+        out.add(norm_uri(u[4:]) or '/')
+    else:
+        out.add(norm_uri('/api' + u))
+    return {x for x in out if x}
+
+
+def load_ledger(path):
+    rows = [l.rstrip('\n').split('\t') for l in open(path, encoding='utf-8') if l.strip()]
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)